src/pipes/is-in-date-range.pipe.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  registerDecorator,
+  ValidationOptions,
+  ValidationArguments,
+} from 'class-validator';
+
+export function IsInDateRange(
+  minDate: string,
+  maxDate?: string,
+  validationOptions?: ValidationOptions,
+) {
+  return function(object: Object, propertyName: string) {
+    registerDecorator({
+      name: 'isInDateRange',
+      target: object.constructor,
+      propertyName: propertyName,
+      options: validationOptions,
+      validator: {
+        validate(value: any, _args: ValidationArguments) {
+          if (value) {
+            const dateObject = new Date(value);
+            const minDateDate = new Date(minDate);
+            const maxDateDate =
+              maxDate !== undefined && maxDate !== null
+                ? new Date(maxDate)
+                : new Date();
+            return dateObject >= minDateDate && dateObject <= maxDateDate;
+          }
+          return true;
+        },
+      },
+    });
+  };
```

Comment:
Since this Pipe is common between the other APIs, I would suggest you to move this to EASEY-Common

Reply:
I agree that this should be done but it is outside the scope of this ticket. Another ticket for this will be created